src/functions-and-arrays.js

```diff
@@ -29,9 +146,53 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(arr) {
+
+  //probar si null
+  if (arr.length === 0) {
+    return null;
+  }
+
+  newArr = [];
+  i = 0;
+  j = 0;
+  doublon = false;
+  while (i<arr.length) {
```

Comment:
Crees que hace falta dos while aca?

Podes hacer un for y chequear algo asi.

if(!arr.slice(i, arr.length).includes(arr[i])) newArr.push(arr[i])